### PR TITLE
add bash

### LINKED ISSUE #15: Install bash
It would be nice to have bash instead of sh only

### PR DESCRIPTION
**Pull request by jomach**
Fixes #15

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,8 @@ RUN set -eux \
 		musl-dev \
 		openssl-dev \
 		python3 \
-		python3-dev
+		python3-dev \
+		bash
```
Comment (cytopia) [inline Dockerfile:12]: @jomach  thanks for the contribution! Any specifics on why `bash` is required for the linting?

Comment (jomach) [reply Dockerfile:12]: For my use case I need to run some more scripts on Jenkins. So I start this image as a slave and run the linting as one of 3 steps. The other steps are bash scripts

Comment (jomach) [reply Dockerfile:12]: For my use case I need to run some more scripts on Jenkins. So I start this image as a slave and run the linting as one of 3 steps. The other steps are bash scripts

Comment (cytopia) [reply Dockerfile:12]: @jomach this is the build image. Anything added here will not be present in the final image. `bash` should be added on line 32

**Comment by cytopia** [comment]
Closing in favour of: https://github.com/cytopia/docker-ansible-lint/pull/18

**Pull request closed without merging**